homeassistant/components/climate/touchline.py

```diff
@@ -0,0 +1,83 @@
+"""
+Demo platform that offers a fake climate device.
+
+Example configuration:
+climate:
+  - platform: touchline
+    host: http://192.168.1.1
+
+For more details about this platform, please refer to the documentation
+https://home-assistant.io/components/climate.touchline/
+"""
+
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.climate import ClimateDevice, PLATFORM_SCHEMA
+from homeassistant.const import CONF_HOST, TEMP_CELSIUS
+import homeassistant.helpers.config_validation as cv
+
+REQUIREMENTS = ['pytouchline==0.5']
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_HOST): cv.string
+})
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Set up the Touchline devices."""
+    from pytouchline import PyTouchline
+    host = config[CONF_HOST]
+    py_touchline = PyTouchline()
+    numberOfDevices = int(py_touchline.get_number_of_devices(host))
```

Comment:
I think home assistant is using `underscore_format` for variables and `CamelCase` for classes.

Reply:
Thanks for your comments. Do you mean CamelCase for objects?

Reply:
No, for classes. Objects are just variables, i.e. use underscore notation.